src/Agent.Sdk/Knob/AgentKnobs.cs

```diff
@@ -152,6 +152,13 @@ public class AgentKnobs
             new EnvironmentKnobSource("AGENT_USE_NODE16"),
             new BuiltInDefaultKnobSource("false"));
 
+        public static readonly Knob UseNode20 = new Knob(
```

Comment:
Thanks for the feedback.  I see this is following the pattern where UseNode16 knob was added.   My thinking is it could be helpful e.g. in case there is a need to force the tasks to run on Node20

Reply:
@KonstantinTyukalov Thanks for comment. I put it here to align with what we had for UseNode16, just in case if this variable will be needed same way as we do for node 16.